src/main/java/com/cognifide/knotx/repository/RepositoryFacade.java

```diff
@@ -34,6 +34,7 @@
     @Override
     public void get(URI uri, AsyncResultHandler<Template<String, URI>> handler) throws IOException {
         repositoryConfiguration.getRepositories().stream()
+                .filter(metadata -> metadata.getType().validate(metadata))
```

Comment:
I have restored 'afterPropertiesSet' method in RepositoryConfiguration class. But instead of throwing exception its logging the fact that configuration is invalid. Is that ok?